packages/vue-renderer/src/renderer.js

```diff
@@ -291,7 +291,26 @@ export default class VueRenderer {
     return fn(opts)
   }
 
-  async renderRoute(url, context = {}, retries = 5) {
+  async renderSPA(context, ENV) {
+    const content = await this.renderer.spa.render(context)
+
+    const APP =
+      `<div id="${this.context.globals.id}">${this.context.resources.loadingHTML}</div>` +
+      content.BODY_SCRIPTS
+
+    const html = this.renderTemplate(false, {
+      ...content,
+      APP,
+      ENV
```

Comment:
Why not giving `this.context.options.env`?

Reply:
Indeed better idea 👍  Funny fact: It was in my last commit which I didn't push before leaving home 😆